### PR TITLE
Add chdb results on c6a.metal

### PR DESCRIPTION
**Pull request by nmreadelf**
[chDB](https://github.com/auxten/chdb) is an in-process SQL OLAP Engine powered by ClickHouse

### CODE REVIEW

**Comment by nmreadelf** [comment]
> pls update `index.html` by running `generate_results.sh`

done

**Comment by nmreadelf** [comment]
this pull request ready for merging.

File: index.html [hunk 1]
```diff
@@ -338,10 +338,10 @@
...
+,{"system":"clickhouse-local (single)","date":"2022-03-19","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"","tags":["C++","column-oriented","embedded","stateless","ClickHouse derivative"],"load_time":0,"data_size":14779976446,"result":[[0.562,0.252,0.259],[0.391,0.153,0.162],[0.922,0.411,0.398],[1.534,0.387,0.359],[2.708,2.198,2.183],[7.544,6.054,6.084],[0.498,0.228,0.243],[0.307,0.196,0.189],[3.701,2.62,2.657],[4.501,3.425,3.429],[2.232,1.502,1.498],[2.446,1.764,1.779],[6.254,4.851,4.725],[7.765,6.045,6.147],[6.939,6.076,6.043],[3.932,3.186,3.103],[16.708,14.158,14.351],[17.05,14.167,14.234],[31.272,27.217,26.692],[1.589,0.371,0.367],[16.325,7.498,8.14],[19.533,9.65,9.603],[37.028,17.607,17.483],[118.034,68.427,68.498],[6.669,3.45,3.422],[3.195,2.507,2.496],[5.387,3.466,3.451],[16.36,7.476,8.029],[84.663,76.769,76.871],[3.755,3.599,3.268],[7.065,4.214,4.187],[9.788,5.538,5.481],[25.194,21.316,20.755],[29.257,19.76,20.61],[28.11,19.46,19.662],[6.841,5.826,5.833],[17.37,7.234,7.215],[16.988,8.204,8.152],[16.611,7.302,7.19],[30.044,13.501,13.521],[5.037,1.505,1.496],[3.22,1.402,1.38],[3.217,1.441,1.454]],"source":"clickhouse-local/results/c6a.4xlarge.single.json"}
```
Comment (nickitat) [inline index.html:344]: why dates changed here?

Comment (nickitat) [reply index.html:344]: oops, we forgot to update index.html here: https://github.com/ClickHouse/ClickBench/pull/91/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051

**Pull request merged**